README.md

```diff
@@ -22,24 +36,37 @@
 >To check $SYSLOG_MON is set: <code>echo $SYSLOG_MON</code>
 
 ### Run docker container from a docker image named syslog-server
+#### By docker cmd
 - Run docker:
   ```
-  docker run --restart unless-stopped --network=host -d -it --name syslog-sv -v $SYSLOG_MON:/etc/syslog-ng/syslog_monitoring/Config/ syslog_monitoring:<tag> bash
+  docker run --rm --network=host --name syslog-sv -v $SYSLOG_MON:/etc/syslog-ng/syslog_monitoring/Config/ syslog_monitoring:<tag>
```

Comment:
If you are not using a service to start the docker container, then you need to add to the docker command line:
`--restart unless-stopped -d`
And you must remove `--rm` to prevent the container disappearing when it is stopped.